lib/database/adapters/SimulationPassAdapter.js

```diff
@@ -52,8 +52,8 @@ class SimulationPassAdapter {
             requestedEventsCount,
             generatedEventsCount,
             outputSize,
-            runsCount,
-            dataPassesCount,
+            runsCount: runs?.[0]?.get('count') ?? null,
+            dataPassesCount: dataPasses?.[0]?.get('count') ?? null,
```

Comment:
As we discussed, it would be great to have another way to get count without depending on `[0]`

Reply:
yes It's refactored
Anyway, I needed to refactor it once again, because this time  I spotted that count wasn't working when filtering with dataPassId because of using  dataPassId condition in outer join clause what affected number of counted data passes associated to given simulation pass